components/BookDetails/BookDetails.js

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { StyledDetailsPage } from "./BookDetails.style";
+import { StyledImage } from "./BookDetails.style";
+import { StyledListDetails } from "./BookDetails.style";
+import { StyledDescription } from "./BookDetails.style";
+
+export default function BookDetails({
+  imageURL,
+  title,
+  author,
+  category,
+  ISBN,
+  EAN,
+  language,
+  publication,
+  pagenumber,
+  price,
+  currencyCode,
+  description,
+}) {
+  const bookDetails = [
+    { label: "Title", value: title },
+    { label: "Author", value: author },
+    { label: "Category", value: category },
+    { label: "ISBN", value: ISBN },
+    { label: "EAN", value: EAN },
+    { label: "Language", value: language },
+    { label: "Publication", value: publication },
+    { label: "Number of pages", value: pagenumber },
+    { label: "Price", value: `${price} ${currencyCode}` },
+  ];
```

Comment:
too much props passed for my taste:

```suggestion
export default function BookDetails({ book }) {
  const bookDetails = [
    { label: "Title", value: book.title },
    { label: "Author", value: book.author },
    { label: "Category", value: book.category },
    { label: "ISBN", value: book.ISBN },
    { label: "EAN", value: book.EAN },
    { label: "Language", value: book.language },
    { label: "Publication", value: book.publication },
    { label: "Number of pages", value: book.pagenumber },
    { label: "Price", value: `${book.price} ${currencyCode}` },
  ];
```